documentation/modules/configuring/con-config-high-volume-messages.adoc

```diff
@@ -0,0 +1,238 @@
+// Module included in the following assemblies:
+//
+// assembly-config-high-volumes.adoc
+
+[id='con-high-volume-config-properties-{context}']
+= Handling high volumes of messages
+
+[role="_abstract"]
+If your Strimzi deployment needs to handle a high volume of messages, you can use configuration options to optimize for throughput and latency. 
+
+Producer and consumer configuration can help control the size and frequency of requests to Kafka brokers. 
+For more information on the configuration options, see the following:
+
+* xref:con-producer-config-properties-throughput-{context}[] 
+* xref:con-consumer-config-properties-throughput-{context}[]  
+
+You can also use the same configuration options with the producers and consumers used by Kafka Connect runtime and MirrorMaker 2.0 source connectors and sink connectors.
+
+Source connectors:: 
+* Consumers retrieve messages from a source Kafka cluster for MirrorMaker 2.0. As Kafka Connect fetches data from an external data system, consumers are not usually required for source connectors.  
```

Comment:
What about:
```
* Producers from the Kafka Connect runtime send messages to the Kafka cluster.
* In case of MirrorMaker, since the source system is Kafka, consumers retrieve messages from a source Kafka cluster
```